lib/helpers.coffee

```diff
@@ -158,6 +158,34 @@ module.exports = Helpers =
         )
     return toReturn
 
+  findFileAsync: (directory, names) ->
+    throw new Error "Specify a filename to find" unless arguments.length
+    unless names instanceof Array
+      names = [names]
+
+    chunks = directory.split(path.sep)
+    promise = Promise.resolve(null)
+
+    while chunks.length
+      currentDir = chunks.join(path.sep)
+      break unless currentDir
+
+      do (currentDir) ->
```

Comment:
You could just write this function in another file and require it in the CoffeeScript file now, so you don't have to rewrite this new function later.